test/MultisigTask.t.sol

```diff
@@ -0,0 +1,249 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Should this file go into the `task/` directory and not sit at the root of the `test/` directory? e.g. lives in the same place as `SingleMultisigTask.t.sol`.